src/Authorization/Controllers/DecisionController.cs

```diff
@@ -52,6 +53,8 @@ public class DecisionController : ControllerBase
         private readonly IAccessManagementWrapper _accessManagement;
         private readonly IMapper _mapper;
 
+        private readonly SortedDictionary<string, AuthInfo> _authInfoCache = new();
```

Comment:
Can you please rename this instanceInfoCache or something so it actually describes what it is.
The AuthInfo model naming is way too vague of a naming for something used in Authorization context.

Reply:
Have renamed authInfoCache to authInfoForOneDecReq

Reply:
wow, that's even worse 😆
My issue is with AuthInfo. It doesn't contain authorization information. It's AppInstanceInfo.
That it's a local cache per. decision request is less relevant, or could be a comment.